### ISSUE TITLE
Accessibility: action buttons not labeled

### ISSUE THREAD

**Issue by shortwavesurfer2009** [PrimalHQ/primal-android-app#246]
## Describe the bug
When using the TalkBack screen reader, buttons such as reply, Zap, like, and bookmark, all say "leading icon" instead of being properly labeled.

### Steps to reproduce
1. Open primal
2. In android settings > accessibility > talkback turn it on
3. Swipe up with 2 fingers and hold to bring up app switcher
4. Swipe right and or left until primal is spoken then double tap with 1 finger to open it
5. Tap the buttons mentioned (video included below)

### Expected behavior
The buttons should be properly labeled.

### Environment
 - primal 2.0.28

### Additional context
Video: https://files.catbox.moe/67bjum.mp4

**Comment by mauritssmedes** [comment]
Can I work on this issue? @AleksandarIlic

**Comment by AleksandarIlic** [comment]
Yeah, sure @mauritssmedes. That would be very helpful, thanks!

Just make sure that all strings starts with "accessibility" prefix.  We already have some in `strings.xml`.
Or maybe it's a good idea to have separate xml file just for "accessibility" strings.

**Comment by mauritssmedes** [comment]
Hi @AleksandarIlic ,

I’ve built the debug and both production builds and noticed that the action buttons already have the correct labels. It seems the app from the Play Store might be missing some code or running a different version. Specifically, it doesn’t display the correct labels as described by @shortwavesurfer2009

This could be caused by a build setting or configuration issue during the Play Store deployment. I’ve tested all three builds on both the emulator and my phone, and they consistently show the correct labels.

Let me know if you’d like me to investigate further or if there are any specific build settings you’d like me to double-check!

**Comment by shortwavesurfer2009** [comment]
Just as an additional data point I got my app from F-Droid. I don't know if that makes any particular difference, but since Play Store was mentioned above, I figured it was worth bringing up just in case.

**Comment by mauritssmedes** [comment]
> Just as an additional data point I got my app from F-Droid. I don't know if that makes any particular difference, but since Play Store was mentioned above, I figured it was worth bringing up just in case.

On my personal phone primal app (play store downloaded) the labels are incorrect too. So it shouldn't matter that you got yours from f-droid

**Comment by AleksandarIlic** [comment]
This is strange. Are the labels correct in APK builds from Github Release pages?

If yes, then it might be something wrong with `https://github.com/Triple-T/gradle-play-publisher` gradle plugin we are using to publish the builds on Google Play.

**Comment by mauritssmedes** [comment]
The labels are incorrect in the primal-2.0.28.apk from the release page

we are probably building the apk in a different way, in which appaearantly i do something that results in the correct labels.

**Comment by AleksandarIlic** [comment]
This is then not related to the `gradle-play-publisher`. It could be that the obfuscation process removes the accessibility strings. You said you tried both debug and production builds, did you tried the build variant ending with `release`? Like `aospAltRelaase` or `googlePlayRelease`? You'll probably need your own key and `config.properties` to build it. There is an explanation on the README page for this.
